### PR TITLE
sorcery: moved import to top

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Moved import stmt from bottom to the top-ish of the file

##### ISSUE TYPE

- Refactoring Pull Request

##### COMPONENT NAME

sorcery

### CODE REVIEW

File: changelogs/fragments/6525-sorcery-import.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - sorcey - minor refactor (https://github.com/ansible-collections/community.general/pull/6525).
```
Comment (vaygr) [inline changelogs/fragments/6525-sorcery-import.yaml:2]:
```suggestion
  - sorcery - minor refactor (https://github.com/ansible-collections/community.general/pull/6525).
```

**Review by vaygr — CHANGES_REQUESTED** [review]

File: changelogs/fragments/6525-sorcery-import.yaml (hunk 1 above)
Comment (russoz) [reply changelogs/fragments/6525-sorcery-import.yaml]: Thanks for spotting it!

**Review by felixfontein — APPROVED** [review]
Will merge for 7.1.0.

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!
@vaygr thanks for reviewing!

**Pull request merged**